somafm.el

```diff
@@ -164,6 +176,25 @@ If the favorite file doesn't exist it will be created."
               (delete-char -1)
               (somafm--clear-rest-of-line))))))))
 
+(defun somafm--volume-control (steps direction)
+  "Modify the volume n STEPS in DIRECTION."
+  (let ((steps (if steps steps 1))
+	(direction-key (if (string= "up" direction)
+			   somafm-mpv-volume-up-key
```

Comment:
if we could make these tabs into spaces please -- not a big deal, but just so it's consistent with the rest of the project